src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,47 @@
+import { ReactNode } from "react";
+import { Icon, IconKey } from "src/components/Icon";
+import { Css, Palette } from "src/Css";
+import { useTestIds } from "src/utils";
+import { IconButton } from "../IconButton";
+import { useToastContext } from "./ToastContext";
+
+export type ToastTypes = "error" | "warning" | "success" | "info" | "alert";
+export interface ToastProps {
+  type: ToastTypes;
+  message: ReactNode;
+}
+
+export function Toast(props: ToastProps) {
+  const { message, type } = props;
+  // using toast context to set notice to undefined once the user clicks out of the alert
+  const { setNotice } = useToastContext();
+  const tid = useTestIds(props, "toast");
+
+  return (
+    <div css={{ ...variantStyles[type], ...Css.df.aic.w100.gap2.px2.py2.$ }} {...tid} role="alert">
+      <span css={Css.fs0.plPx(12).$}>
+        <Icon icon={typeToIcon[type]} {...tid.type} color={Palette.Gray900} />
+      </span>
+      <span {...tid.message}>{message}</span>
+      <span css={Css.pr1.add("lineHeight", 0).add("marginLeft", "auto").$}>
+        <IconButton icon="x" contrast onClick={() => setNotice(undefined)} {...tid.close} color={Palette.Gray900} />
```

Comment:
**nit:** I don't think we want `contrast` here. It makes an odd hover state...
<img width="92" alt="Screen Shot 2022-11-09 at 10 38 51 AM" src="https://user-images.githubusercontent.com/1143861/200885282-df79fb2f-1128-4440-9cbd-d12fb0db26ab.png">